src/elements/visual-checkbox/visual-checkbox.stories.ts

```diff
@@ -121,6 +130,30 @@ export const DisabledIndeterminateNegative: StoryObj = {
   args: { ...defaultArgs, indeterminate: true, disabled: true, negative: true },
 };
 
+export const SizeXS: StoryObj = {
+  render: Template,
+  argTypes: defaultArgTypes,
+  args: { ...defaultArgs, size: size.options![2] },
+};
+
+export const SizeXSChecked: StoryObj = {
+  render: Template,
+  argTypes: defaultArgTypes,
+  args: { ...defaultArgs, size: size.options![2], checked: true },
+};
+
+export const SizeXSIndeterminate: StoryObj = {
+  render: Template,
+  argTypes: defaultArgTypes,
+  args: { ...defaultArgs, size: size.options![2], indeterminate: true },
+};
+
+export const SizeXSNegative: StoryObj = {
+  render: Template,
+  argTypes: defaultArgTypes,
+  args: { ...defaultArgs, size: size.options![2], negative: true },
+};
+
```

Comment:
I'm conflicted here :|
We don't need combinations of params in stories anymore but maybe we should remove them once we definitively abandon chromatic.
What do you think?

Reply:
We'll remove chromatic in 6 months if I remember correctly; 
in the meantime i think we should continue to provide examples to users